python/lsst/ts/ofc/ofc_data/ofc_data.py

```diff
@@ -350,122 +314,207 @@ def _configure_instrument(self, value):
                     with open(path) as fp:
                         self.bend_mode[comp][ftype]["data"] = yaml.safe_load(fp)
 
-        self.log.debug(f"Configuring {value}")
-
-        inst_config_dir = self.config_dir / value
-        if not inst_config_dir.exists():
-            RuntimeError(
-                f"Instrument {inst_config_dir!s} configuration directory does not exists. "
-                f"Make sure the name you are passing ({value}) exists in the configuration directory: "
-                f"{self.config_dir!s}."
-            )
+        self.log.debug(f"Configuring {instrument}")
 
         # Load all data to local variables and only set them at the end if
         # everthing went fine. Otherwise you can leave the class in a broken
         # state.
 
         # Read dof_state0
-        dof_state0_path = inst_config_dir / self.dof_state0_filename
+        # ---------------
+        dof_state0_path = self.config_dir / self.dof_state0_filename
 
         with open(dof_state0_path) as fp:
             dof_state0 = yaml.safe_load(fp)
 
-        # Read image quality weight
-        iqw_path = inst_config_dir / self.iqw_filename
+        # Read sample points
+        # ------------------
+        sample_points_path = (
+            self.config_dir / "sample_points" / f"{instrument}_points.yaml"
+        )
+
+        self.log.debug(f"Configuring sample points: {sample_points_path}")
+        try:
+            with open(sample_points_path) as fp:
+                sample_points = yaml.safe_load(fp)
+        except FileNotFoundError:
+            raise RuntimeError(
```

Comment:
Please add this RuntimeError to the function doc string. BTW, you had used some `try` loop to read yaml file as a duplication. Please write this part as a function to reuse this part of logic.